src/ToDoList.js

```diff
@@ -1,71 +1,34 @@
+import {
+  isEmpty, markUpTask, saveLocalStorage, retrieveLocalStorage, cloneCollection, taskObj,
+} from './Utils.js';
+
 class ToDoList {
   constructor() {
     this.tasksList = [];
   }
```

Comment:
- [Optional] Your app throughs an exception on startup mentioning  `Uncaught TypeError: collection is not iterable`, I suspect this might have to do with the startup loading of data. You might have some local storage already in place which makes the app work for you, In my can I could not add a task at all.   

![image](https://github.com/NitBravoA92/project-to-do-list/assets/69077061/9637c51c-6b6f-4706-ad16-3ebe0cb8cd80)